src/Microsoft.IdentityModel.JsonWebTokens/JsonWebToken.cs

```diff
@@ -127,9 +126,12 @@ public virtual IEnumerable<Claim> Claims
         {
             get
             {
-                List<Claim> claims = new List<Claim>();
+                var claims = new List<Claim>();
                 string issuer = this.Issuer ?? ClaimsIdentity.DefaultIssuer;
 
+                if (Payload == null)
+                    return claims;
```

Comment:
>claims [](start = 27, length = 6)

Allocating a new List can be avoided in this scenario by moving the IF check above where 'claims' is initialized, and instead returning Enumerable.Empty<Claim>()

Reply:
Good suggestion. Thanks! 👍